### PR TITLE
small fixes for Swedac!!

### PR DESCRIPTION
**Pull request by senthil10**
Again a small **PR** for fixing some minor stuffs.
Tack @pekrau for the magic bean (`to_acsii`) :)

### CODE REVIEW

**Comment by ewels** [comment]
Nice, looks good!

**Comment by senthil10** [comment]
Tack @ewels :+1:

File: ngi_reports/stockholm/project_summary.py [hunk 1]
```diff
@@ -36,6 +37,13 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 self.LOG.error("No project name found - please specify using '--project'")
                 raise KeyError("No project name found - please specify using '--project'")
         
+        ## Check and exit if signature not provided
+        if not kwargs.get('signature'):
+            self.LOG.error("It is required to provide Signature/Name generating 'project_summary' report")
```
Comment (ewels) [inline ngi_reports/stockholm/project_summary.py]: For the future - there's a logging level called `exception` which may be better here: https://docs.python.org/2/library/logging.html#logging.Logger.exception

(or `critical` at least)

Comment (senthil10) [inline ngi_reports/stockholm/project_summary.py]: Tack for the info @ewels , will look into it and try to implement in the future PR :+1:

**Pull request merged**